proposals/p2138.md

```diff
@@ -0,0 +1,51 @@
+# Checked and template generic terminology
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/2138)
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Abstract](#abstract)
+-   [Problem](#problem)
+-   [Proposal](#proposal)
+-   [Rationale](#rationale)
+
+<!-- tocstop -->
+
+## Abstract
+
+Change terminology from "generics" and "templates" to "checked generics" and
+"template generics", so the term "generics" will now include templates refers to
```

Comment:
This doesn't quite scan. Perhaps `... so the term "generics" now refers to both use cases`?

Reply:
You are correct. I rewrote to use the word "umbrella". Better now?